requirements.txt

```diff
@@ -1,5 +1,5 @@
 jsonschema
-openapi-schema-validator
+git+https://github.com/anexia-it/openapi-schema-validator@master
```

Comment:
This should be reverted once https://github.com/p1c2u/openapi-schema-validator/pull/18 is completed and a new release is available

Reply:
updated to `openapi-schema-validator>=0.3.0a1`